jobs/delorean/2.0/ews/3scale/discovery.yaml

```diff
@@ -22,6 +22,9 @@
     - string:
           name: CI_MESSAGE
           description: This will be populated by the CI trigger
+    - string:
+          name: PRODUCT_BRANCH
+          description: The name of the product branch
```

Comment:
How does this one get populated?

Reply:
Ah yeah, forgot to default it. I'll add that in